cmd/mirror-main.go

```diff
@@ -1011,6 +1015,7 @@ func runMirror(ctx context.Context, srcURL, dstURL string, cli *cli.Context, enc
 		isMetadata:            isMetadata,
 		isSummary:             cli.Bool("summary"),
 		isRetriable:           cli.Bool("retry"),
+		failOnError:           cli.Bool("fail-on-error"),
```

Comment:
You must add this to the [flags](https://github.com/minio/mc/blob/391616f9cfb9af5799a5cea63ace1a48b4c11481/cmd/mirror-main.go#L49) as well AFAICT.

Reply:
Added in [cf8fbed](https://github.com/minio/mc/pull/5163/commits/cf8fbed54fefdb34279076029284c855ade7da16)